uiapp.rb

```diff
@@ -1,26 +1,101 @@
+require_relative './controller/authoractions'
+require_relative './controller/labelactions'
+require_relative './controller/bookactions'
+require_relative './controller/genre_actions'
+require_relative './controller/music_album_actions'
+require_relative './controller/gameactions'
 class UiApp
-  def initialize(books, music_albums, games)
-    @games = RentalActions.new(games)
-    @music_albums = PersonActions.new(music_albums)
+  def initialize(books, music_albums, games, authors, genres, labels)
+    @games = GameActions.new(games)
+    @labels = LabelActions.new(labels)
+    @authors = AuthorActions.new(authors)
     @books = BookActions.new(books)
+    @genres = GenreActions.new(genres)
+    @music_albums = MusicAblumActions.new(music_albums)
   end
 
   def options
     puts 'Please choose an option by entering a number:'
     puts '0 - Exit'
     puts '1 - List all books'
-    puts '2 - List all labels (e.g. Gift, New)'
-    puts '3 - Add a book'
+    puts '2 - Create a book'
+    puts '3 - List all labels (e.g. Gift, New)'
+    puts '4 - Create Label'
+    puts '5 - List all genres'
+    puts '6 - Create genre'
+    puts '7 - List all music albums'
+    puts '8 - Create Music album'
+    puts '9 - List all Authors'
+    puts '10 - Create Author'
+    puts '11 - List all Games'
+    puts '12 - Create Game'
```

Comment:
- [ ] according to the requirements the application should be able to:
- List all books
- List all music albums
- List of games
- List all genres (e.g 'Comedy', 'Thriller')
- List all labels (e.g. 'Gift', 'New')
- List all authors (e.g. 'Stephen King')
- Add a book
- Add a music album
- Add a game
- Exit
the idea behind this is that when adding a book you should put in an author for example and that's how the authors get created, this should be connected.